app/main/views/users.py

```diff
@@ -35,7 +35,15 @@ def auth_user():
         # will remove camel case email address with future api
         email_address = json_payload.get('emailAddress', None)
 
-    user = User.get_by_email_address(email_address.lower())
+    user = User.query.options(
+        joinedload('supplier'),
+        noload('supplier.*'),
+        joinedload('application'),
+        noload('application.*'),
+        noload('*')
```

Comment:
Does this override previous * ? https://docs.sqlalchemy.org/en/rel_1_1/orm/loading_relationships.html?highlight=noload#wildcard-loading-strategies

Reply:
double checked and it return supplier without child properties